.github/workflows/rest_api_tests.yml

```diff
@@ -81,11 +81,15 @@ jobs:
           pip install .
 
       - name: Run tests
+        env:
+          PYTEST_PARAMS: --maxfail=5 --durations=10 --suppress-no-test-exit-code
         run: |
           pytest ${{ env.PYTEST_PARAMS }} rest_api/
 
       - uses: act10ns/slack@v2
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
           status: ${{ job.status }}
           channel: "#haystack"
-        if: failure() && github.repository_owner == 'deepset-ai' && github.ref == 'refs/heads/main'
```

Comment:
Why do we remove the check for `github.repository_owner == 'deepset-ai'` here?

Reply:
It's not really necessary, if somebody forks the repo no message will be sent in any case since the `SLACK_WEBHOOK_URL` won't be set.

It just complicates the logic having that condition and it has practically no benefit.